src/site/content/en/blog/declarative-link-capturing/index.md

```diff
@@ -0,0 +1,222 @@
+the set of URLs to which an application context can be navigated while the manifest is applied. If
+the `"scope"` member is not present in the manifest, it defaults to the parent path of the
+`"start_url"` member. {% endAside %}
+
+Some special conditions like middle-clicking a link (or right-clicking and then "open in new tab")
+would typically not trigger the link capturing behavior. Whether a link is `target=_self` or
+`target=_blank` does not matter, so that links clicked in a browser window (or window of a different
+PWA) would be opened in the PWA even if they would normally cause a navigation within the same tab.
+
+## Suggested use cases
+
+Examples of sites that may use this API include:
+
+- PWAs that want to open a window, rather than a browser tab, when the user clicks on a link to
+  them. In a desktop environment, it often makes sense to have multiple application windows open at
+  a time.
+- Single-window PWAs where the developer prefers to only have a single instance of the app open at
+  any time, with new navigations focusing the existing instance. Sub-use cases include:
+  - Apps that generally only make sense to have one instance running (e.g., a music player, a game).
+  - Apps that include multi-document management within a single instance (e.g., an HTML-implemented
+    tab strip).
+
+## Current status {: #status }
+
+<div class="w-table-wrapper">
+
+| Step                                     | Status                               |
+| ---------------------------------------- | ------------------------------------ |
+| 1. Create explainer                      | [Complete][explainer]                |
+| 2. Create initial draft of specification | [In Progress][spec]                  |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback)             |
+| **4. Origin trial**                      | **[In progress][ot]** (on Chrome OS) |
+| 5. Launch                                | Not started                          |
+
+</div>
+
+### Enabling via about://flags
+
+To experiment with Declarative Link Capturing locally, without an origin trial token, enable the
+`#enable-desktop-pwas-link-capturing ` flag in `about://flags`.
+
+### Enabling support during the origin trial phase
+
+Starting in Chromium&nbsp;91, Declarative Link Capturing will be available as an origin trial in
+Chromium. The origin trial is expected to end in Chromium&nbsp;94 (October 13, 2021).
+
+{% Aside 'caution' %} The origin trial will initially be limited to the Chrome&nbsp;OS platform.
+{% endAside %}
+
+{% include 'content/origin-trials.njk' %}
+
+### Register for the origin trial {: #register-for-ot }
+
+{% include 'content/origin-trial-register.njk' %}
+
+## How to use Declarative Link Capturing? {: #use }
+
+Developers can declaratively determine how links should be captured by leveraging the additional Web
+Application Manifest field `"capture_links"`. It takes a string or a an array of strings as its
```

Comment:
```suggestion
Developers can declaratively determine how links should be captured by leveraging the additional web
app manifest field `"capture_links"`. It takes a string or a an array of strings as its
```

Reply:
>leveraging

The definition of this is "to use to maximum advantage". Writers tend to use it as a synonym for 'use'. Decided which meaning you intended and use the appropriate word.

Reply:
In this case I mean "to use to maximum advantage". Not just "use".